client/my-sites/marketing/index.js

```diff
@@ -24,13 +25,14 @@ export default function() {
 		'/marketing',
 		'/marketing/connections',
 		'/marketing/sharing-buttons',
+		'/marketing/traffic',
 		'/sharing',
 		'/sharing/buttons',
 	].forEach( path => page( path, ...[ siteSelection, sites, makeLayout, clientRender ] ) );
 
-	page( '/sharing/:domain', redirectConnections );
+	page( '/sharing/:domain', redirectTraffic );
```

Comment:
@belcherj: why did we decide to redirect users who access the former https://wordpress.com/sharing to https://wordpress.com/marketing/traffic? We should redirect them to https://wordpress.com/marketing/connections instead in my opinion.